ajax/libs/slickgrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "slickgrid",
+  "version": "2.3.4",
+  "description": "A lightning fast JavaScript grid/spreadsheet",
+  "filename": "slick.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/6pac/SlickGrid.git"
+  },
+  "keywords": [
+    "slickgrid",
+    "grid"
+  ],
+  "author": {
+    "name": "Michael Leibman",
+    "email": "michael.leibman@gmail.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/mleibman/SlickGrid/wiki",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/6pac/SlickGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "lib/jquery.event.*",
```

Comment:
I think the library `jquery.event.drag` is this one `https://github.com/devongovett/jquery.event.drag`
or this one
`https://github.com/threedubmedia/jquery.threedubmedia`
So I think we don't need to include `lib/jquery.event.*`